examples/gym/train_ppo_batch_gym.py

```diff
@@ -0,0 +1,240 @@
+                self.obs_filter = links.EmpiricalNormalization(
+                    shape=obs_size
+                )
+
+    def pi_and_v(self, state):
+        if self.normalize_obs:
+            state = F.clip(self.obs_filter(state, update=False),
+                           -5.0, 5.0)
+
+        return self.pi(state), self.v(state)
+
+
+def main():
+    import logging
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--gpu', type=int, default=0)
+    parser.add_argument('--env', type=str, default='Hopper-v1')
+    parser.add_argument('--num-envs', type=int, default=1)
+    parser.add_argument('--arch', type=str, default='FFGaussian',
+                        choices=('FFSoftmax', 'FFMellowmax',
+                                 'FFGaussian'))
+    parser.add_argument('--normalize-obs', action='store_true')
+    parser.add_argument('--bound-mean', action='store_true')
+    parser.add_argument('--seed', type=int, default=0,
+                        help='Random seed [0, 2 ** 32)')
+    parser.add_argument('--outdir', type=str, default='results',
+                        help='Directory path to save output files.'
+                             ' If it does not exist, it will be created.')
+    parser.add_argument('--steps', type=int, default=10 ** 6)
+    parser.add_argument('--eval-interval', type=int, default=10000)
+    parser.add_argument('--eval-n-runs', type=int, default=10)
+    parser.add_argument('--reward-scale-factor', type=float, default=1e-2)
+    parser.add_argument('--standardize-advantages', action='store_true')
+    parser.add_argument('--render', action='store_true', default=False)
+    parser.add_argument('--lr', type=float, default=3e-4)
+    parser.add_argument('--weight-decay', type=float, default=0.0)
+    parser.add_argument('--demo', action='store_true', default=False)
+    parser.add_argument('--load', type=str, default='')
+    parser.add_argument('--save-training', action='store_true')
+    parser.add_argument('--logger-level', type=int, default=logging.DEBUG)
+    parser.add_argument('--monitor', action='store_true')
+    parser.add_argument('--max-deque', type=int, default=100)
+
+    parser.add_argument('--update-interval', type=int, default=2048)
+    parser.add_argument('--log-interval', type=int, default=1000)
+    parser.add_argument('--batchsize', type=int, default=64)
+    parser.add_argument('--epochs', type=int, default=10)
+    parser.add_argument('--entropy-coef', type=float, default=0.0)
+    args = parser.parse_args()
+
+    logging.basicConfig(level=args.logger_level)
+
+    # Set a random seed used in ChainerRL
+    misc.set_random_seed(args.seed, gpus=(args.gpu,))
+
+    args.outdir = experiments.prepare_output_dir(args, args.outdir)
+
+    def make_env(test):
+        env = gym.make(args.env)
+        # Use different random seeds for train and test envs
+        env_seed = 2 ** 32 - 1 - args.seed if test else args.seed
+        env.seed(env_seed)
```

Comment:
Each env in VectorEnv should be assigned different random seeds. See `train_a3c_gym.py` for how to assign different random seeds.